Author: Joachim Reichel <reichel@debian.org>
Description: Improve autodetection of thirdparty libraries.
Index: cgal-5.0/cmake/modules/FindGLPK.cmake
===================================================================
--- cgal-5.0.orig/cmake/modules/FindGLPK.cmake
+++ cgal-5.0/cmake/modules/FindGLPK.cmake
@@ -15,7 +15,7 @@ if (NOT GLPK_FOUND)
              )
 
     find_library(GLPK_LIBRARIES
-                 NAMES libglpk
+                 NAMES glpk libglpk
                  PATHS ENV LD_LIBRARY_PATH
                        ENV LIBRARY_PATH
                        /usr/local/lib
Index: cgal-5.0/demo/Polyhedron/Plugins/IO/CMakeLists.txt
===================================================================
--- cgal-5.0.orig/demo/Polyhedron/Plugins/IO/CMakeLists.txt
+++ cgal-5.0/demo/Polyhedron/Plugins/IO/CMakeLists.txt
@@ -80,6 +80,7 @@ endif()
 
 find_path(3MF_INCLUDE_DIR
   NAMES Model/COM/NMR_DLLInterfaces.h
+  PATHS /usr/include/lib3mf
   DOC "Path to lib3MF headers"
   )
 find_library(3MF_LIBRARIES NAMES 3MF DOC "Path to the lib3MF library")
